code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm

```diff
@@ -34,6 +36,16 @@
 	else
 		..()
 
+/mob/living/simple_animal/hostile/megafauna/proc/grant_achievement()
```

Comment:
I don't think we actually track who has been hitting what currently (other than writing something new to interpret attack logs?)